lib/src/scenario_generator.dart

```diff
@@ -13,18 +13,25 @@ void parseScenario(
 ) {
   sb.writeln(
       '    $testMethodName(\'\'\'$scenarioTitle\'\'\', (tester) async {');
+  sb.writeln('      try {');
   if (hasSetUp) {
-    sb.writeln('      await $setUpMethodName(tester);');
+    sb.writeln('        await $setUpMethodName(tester);');
   }
 
   for (final step in scenario) {
-    sb.writeln('      await ${getStepMethodCall(step.value)};');
+    sb.writeln('        await ${getStepMethodCall(step.value)};');
   }
 
+  sb.writeln('      } on Exception catch (error, stackTrace) {');
+  sb.writeln('        debugPrint(\'\${error.toString()}: \$stackTrace\');');
+  sb.writeln('        rethrow;');
```

Comment:
Please remove the catch block and keep `try..finally` only. For debugging purposes, it is beneficial to print exceptions, but this is not always needed.